3_Automation/06_Bestiary.yml

```diff
@@ -43,45 +43,30 @@ Name:                                 # must be unique
   Type: Companion, level 1
```

Comment:
Treating all creatures, including PCs as the same will result in some issues differentiating Eagle level 1 from level 2. If I want to look up Eagle level 2, where would that be? Maybe `Eagle 1` needs to be creature name or 
```yaml
Eagle:
    Level1:
        HP: 1 (and other stats)
    Level2: 
        HP: 2

Reply:
I think the solution here is for unique names that indicate power/strength as opposed to level. 
Wolf --> Dire Wolf 
Falcon --> Bladed Falcon 

Or we give a stat boost on level up in certain cases without changing the name, like adding a formula to Companion HP and +to hit: 
+ Companion HP = 1+ (Primary Skill Mod/2)  
+ Companion to hit = 0 + Primary Skill Mod